### ISSUE TITLE
Intune SDK coexisting with other MDM SDK's

### ISSUE THREAD

**Issue by Abhinay1997** [msintuneappsdk/ms-intune-app-sdk-ios#177]
Hello Team,

#### Environment: -iOS 13+ , Intune 14.4.1, Apple App Store deployment planned.

We are currently working on an iOS app that uses Intune SDK and MSAL for MAM/MDM and Login respectively.

A new requirement is that the app also support other MDM SDK's like Workspace1, Citrix endpoint management etc.. I was wondering if multiple MDM SDK's can coexist and Intune or another SDK be triggered at runtime based on the login credentials ( this will be handled by the app )

#### Specifically:-

1. If I use MSAL for user login but in Info.plist set: "AutoEnrollOnLaunch: false, MAMPolicyRequiredOnLaunch: false", don't call loginAndEnroll or registerAndEnroll, will the SDK still try to check for app policy and try to apply any policies tied to that user account ?

#### Basically 2 types of users:-

User1: Enrolls with workspace1 SDK, fetches the App Config for AzureAD url, Authority for his organization. MSAL uses that to login to the application.

User2: Enrolls with Intune SDK, the required app config policy is fetched and also logs him into the application. ( loginAndEnroll or MSAL authentication followed by registerAndEnroll) 

I referred to #32 and #176 and I believe Intune shouldn't cause any issues. Wanted to know your thoughts @Kyle-Reis

**Comment by Kyle-Reis** [comment]
Hi @Abhinay1997 - If no Intune enrollment method is invoked, or silent enrollment (registerAndEnrollAccount) fails for any reason, including the user isn't licensed for Intune or targeted with policy, then Intune should not have any effect on app behavior.

**Comment by Abhinay1997** [comment]
Hi Kyle, Thank you for clarifying this. Closing the issue.
